README.md

```diff
@@ -1 +1,17 @@
-Bootstrap languages
\ No newline at end of file
+Bootstrap Languages
+===================
+Bootstrap module integrates the
+[Bootstrap Languages](https://github.com/usrz/bootstrap-languages)
+
+## Installation for Drupal 8
+
+Download and install the 
+[Bootstrap Languages]
+(https://github.com/usrz/bootstrap-languages/archive/master.zip)
+
+Unpack and rename the Bootstrap Languages library directory to 
+`bootstrap-languages` and place it inside the `/libraries` 
```

Comment:
I think here the path should be different 
look at https://www.drupal.org/docs/8/modules/libraries-api-8x/using-libraries-api-3x-as-a-module-developer

> You should instruct your users to place the external library sources in a location within their local Drupal file  structure that Libraries API is aware of, typically /sites/all/assets/vendor/[lib_machine_name] (for asset libraries) or /sites/all/libraries/[lib_machine_name] (for PHP libraries). You can add notes about this to your module's documentation or simply direct users to the Installing an external library that is required by a contributed module page of the Libraries API documentation. Of course, you will also need to ensure that library sources are appropriately located on any development environments that you are using.